src/main/include/subsystems/Flywheel.hpp

```diff
@@ -115,10 +115,13 @@ class Flywheel : public ControllerSubsystemBase {
     frc::Encoder m_encoder{Constants::Flywheel::kEncoderA,
                            Constants::Flywheel::kEncoderB};
     FlywheelController m_controller;
+    std::chrono::steady_clock::time_point m_time;
     std::chrono::steady_clock::time_point m_lastTime =
         std::chrono::steady_clock::now();
     frc::Pose2d m_nextTurretPose;
     frc2::Timer m_timer;
+    units::radian_t m_angle;
+    units::radian_t m_lastAngle;
```

Comment:
Move these to after m_controller and before m_time so all the relevant variables are grouped together.